integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/l0promotion/DistroXOSUpgradeTests.java

```diff
@@ -86,6 +86,7 @@ public void testBaseDistroXOSUpgrade(TestContext testContext) {
                 .given(SdxUpgradeTestDto.class)
```

Comment:
Just a question: doesn't setting the skipBackup flag in the entity's valid() method have an effect on all the usages of the entity? I.e., do you really need to set it separately? Just asking, I am not that familiar with the code, but just checking what given() does, it seems that it sets the default behavior.